config/config.go

```diff
@@ -47,7 +47,7 @@ var (
 		Version:        2,
 		MaxRepetitions: 25,
 		Retries:        3,
-		Timeout:        time.Second * 20,
+		Timeout:        time.Second * 2,
```

Comment:
I'd suggest leaving this a little bit higher, maybe 5s. Some devices/networks are particularly slow.

Reply:
For a single GETBULK that's close on the network?

There is also an exponential option (which is what broke us) now that we could use.

Reply:
This is also a default, so if there's slow devices they could be configured by with a longer value without slowing down everyone who only loses a packet.

Reply:
Yes, I think the default should be relatively relaxed for this kind of thing.

Reply:
Changed.